draft-ietf-core-problem-details.md

```diff
@@ -761,6 +761,15 @@ supplied for its initial registration.
 It has been extended to allow supplementing the language tag by a
 direction indication.
 
+As with any IANA-registered item, a specification that further updates this
+registration needs to update the reference column of the IANA registry (see
+{{iana-tag38}}).
+Future specifications may update this appendix, other parts of this document,
+or both.
```

Comment:
To cover all comments I saw on the mailing list, can we add something about future updates not dropping part of the current text? (for example leaving the appendix out)

suggestion: 
or both. Updates to this document need to be clear about what is being altered, and ought not discard this appendix without publishing it as a separate document.

Reply:
I understand why we may want to add the first part of the sentence, but the second is a bit unclear: if the Tag38 part is not altered by the update, the reference in the CBOR tag registry just stays where it is and we should be OK, no?

Reply:
> the second part is about getting into the situation where the main body is updated, and the appendix is discarded, with the reference for the IANA registry pointing to the old document... there were [comments](https://mailarchive.ietf.org/arch/msg/last-call/N3pk_H_z8XyQkx7K0aHrdSb1o9Y/) about that creating document evolution problems. Just trying to cover all basis here.

Sure, and thank you very much for the extreme care you are using in treating this matter!

My point about the second half of your suggestion is that if a future RFC drops the Tag38 appendix and updates PD-specific content, as long as we add the first half of your edit we should be covered because the future RFC will say exactly and clearly what has been updated (which can't be tag38 if the appendix is dropped).  In this case the pointer in the CBOR tags registry will not move.

What we'd be saying if we also added your second half (at least as I read it) is: when/as-soon-as the PD RFC is updated and the appendix is dropped, the Tag38 appendix needs to be split out into its own RFC.  Which seems unnecessary because if nothing has changed the existing reference is still valid.

What am I missing?

Reply:
> > the second part is about getting into the situation where the main body is updated, and the appendix is discarded, with the reference for the IANA registry pointing to the old document... there were [comments](https://mailarchive.ietf.org/arch/msg/last-call/N3pk_H_z8XyQkx7K0aHrdSb1o9Y/) about that creating document evolution problems. Just trying to cover all basis here.
> 
> We did this already with Tag 35. There is nothing wrong with this situation.

There's quite a bit wrong regarding the situation around Tag 35. For details, please see https://mailarchive.ietf.org/arch/msg/art/QaUNS5z4PR_-MDAywpcMDgTvnZ8. If the newer document is just an *update* to the older one, things should be fine, but not if it claims to *obsolete* the older one.

Reply:
I wrote:
> There's quite a bit wrong regarding the situation around Tag 35. For details, please see https://mailarchive.ietf.org/arch/msg/art/QaUNS5z4PR_-MDAywpcMDgTvnZ8. If the newer document is just an _update_ to the older one, things should be fine, but not if it claims to _obsolete_ the older one.
but it turned out I was wrong. Very sorry for the misunderstanding. Please ignore that comment.